package.json

```
@@ -37,13 +37,15 @@
   },
   "scripts": {
     "start:docker": "cd contract && docker compose up -d",
+    "stop:docker": "cd contract && docker compose down",
     "docker:logs": "cd contract; docker compose logs --tail 200 -f",
     "docker:bash": "cd contract; docker compose exec agd bash",
     "docker:make": "cd contract; docker compose exec agd make -C /workspace/contract",
     "make:help": "make -C contract list",
     "start:contract": "cd contract && yarn start",
     "print-key": "yarn docker:make print-acct",
     "start:ui": "cd ui && yarn dev",
+    "start:interchain-ui": "cd interchain-ui && yarn dev",
```

why keep `start:ui`?

```suggestion
    "start:ui": "cd interchain-ui && yarn dev",
```

likewise the `interchain-ui` directory should replace `/ui/`, right?